cmd/ostracon/commands/show_validator.go

```diff
@@ -15,18 +18,29 @@ var ShowValidatorCmd = &cobra.Command{
 	Use:     "show-validator",
 	Aliases: []string{"show_validator"},
 	Short:   "Show this node's validator info",
-	RunE:    showValidator,
-	PreRun:  deprecateSnakeCase,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return showValidator(cmd, args, config)
+	},
+	PreRun: deprecateSnakeCase,
 }
 
-func showValidator(cmd *cobra.Command, args []string) error {
-	keyFilePath := config.PrivValidatorKeyFile()
-	if !tmos.FileExists(keyFilePath) {
-		return fmt.Errorf("private validator file %s does not exist", keyFilePath)
+func showValidator(cmd *cobra.Command, args []string, config *cfg.Config) error {
```

Comment:
Referencing the global variable `root.config` directly is inadvisable because it sometimes causes race conditions when tests are run. By passing it as a parameter, it can be made to work with independent configurations for each test case. Although this way is a test-driven requirement, I think this is a better way from the perspective of the degree of coupling.

Reply:
Got it.